tests/Model_Calibration_v2.py

```diff
@@ -0,0 +1,154 @@
+# ---------
+# AUTHOR: Atul Agrawal (atul.agrawal@tum.de)
+# ---------
+
+import unittest
+import numpy as np
+
+from bayes.parameters import ParameterList
+from bayes.inference_problem import VariationalBayesProblem, ModelErrorInterface
+from bayes.Calibration import Inference
+
+import torch as th
+
+
+class Sensor:
+    def __init__(self, name, shape=(1, 1)):
+        self.name = name
+        self.shape = shape
+
+    def __repr__(self):
+        # return f"{self.name} {self.shape}"
+        return f"{self.name}"
+
+
+class MySensor(Sensor):
+    def __init__(self, name, position):
+        super().__init__(name, shape=(1, 1))
+        self.position = position
+
+
+class MyForwardModel:
+    def __call__(self, parameter_list, sensors, time_steps):
+        """
+        evaluates
+            fw(x, t) = A * x + B * t
+        """
+        A = parameter_list["A"]
+        B = parameter_list["B"]
+
+        result = {}
+        for sensor in sensors:
+            result[sensor] = A * sensor.position + B * time_steps
+        # result=th.reshape(th.tensor(list(result.values())),(-1,))
+        # result=list(result.values())
+        return result
+
+    def parameter_list(self):
+        p = ParameterList()
+        p.define("A", None)
+        p.define("B", None)
+        return p
+
+
+# --- Wrapper for the forward solve class
+def wrapper_forward(known_input, latent_para):
```

Comment:
why do we need a wrapper here?

Reply:
This would go in the model error interface.